docs/getting-started/rbac/admin-role.yaml

```diff
@@ -20,6 +19,7 @@ rules:
   - pipelineruns
   - pipelineresources
   verbs:
+  - get
```

Comment:
I'm not super familiar with the Getting Started guide, but I don't think that we need `- get` here, because I think that we only need to be able to `create` PipelineRuns and PipelineResources.